examples/bert_style_qa/handler.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import sys
+
+import torch
+from transformers import (AutoConfig, AutoTokenizer,
+                          AutoModelForQuestionAnswering)
+
+from dynalab.handler.base_handler import BaseDynaHandler
```

Comment:
Hmmm is this the handler you'll get from the template? Where is `ROOTPATH`? What if people need to import something from their own directory? https://github.com/facebookresearch/dynalab/blob/master/dynalab/handler/handler.py.template#L19

Reply:
I deleted ROOTPATH because it is not used in my handler. If you'd prefer, I could add a comment like this:

```
# You can import ROOTPATH from dynalab.handler.base_handler in the
# circumstance where you want to import code from a directory in your
# project, following the dynalab README.
```